### PR TITLE
Update to asp net core 2.2

### PR DESCRIPTION
**Pull request by RodrigoPereyraDiaz**
This include changes from #16

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 os: Visual Studio 2017
 
-version: 1.2.2-alpha-{build}
+version: 2.1.0-alpha-{build}
```
Comment (RodrigoPereyraDiaz) [inline appveyor.yml]: Make sense keep this as alpha?

File: src/MakingSense.AspNetCore.Documentation/MakingSense.AspNetCore.Documentation.csproj [hunk 2]
```diff
@@ -3,36 +3,27 @@
...
     <AssemblyName>MakingSense.AspNetCore.Documentation</AssemblyName>
     <PackageId>MakingSense.AspNetCore.Documentation</PackageId>
     <PackageTags>markdown;ASP.NET 5;vnext;documentation</PackageTags>
     <PackageProjectUrl>https://github.com/MakingSense/aspnet-documentation-middleware</PackageProjectUrl>
     <PackageLicenseUrl>http://www.gnu.org/licenses/lgpl.html</PackageLicenseUrl>
     <RepositoryType>git</RepositoryType>
     <RepositoryUrl>git://github.com/MakingSense/aspnet-documentation-middleware</RepositoryUrl>
-    <RuntimeFrameworkVersion Condition=" '$(TargetFramework)' == 'netcoreapp1.1' ">1.1.1</RuntimeFrameworkVersion>
-    <VersionPrefix>1.2.2-alpha</VersionPrefix>
+    <VersionPrefix>2.1.0-alpha</VersionPrefix>
```
Comment (RodrigoPereyraDiaz) [inline src/MakingSense.AspNetCore.Documentation/MakingSense.AspNetCore.Documentation.csproj]: Make sense keep this as alpha?

**Review by RodrigoPereyraDiaz — COMMENTED** [review]
Make sense keep version as alpha?

File: appveyor.yml (hunk 1 above)
Comment (andresmoschini) [reply appveyor.yml]: I answer this question in another PR, let us talk about it if you need.

**Comment by andresmoschini** [comment]
Nice!

**Pull request merged**